flexget/plugins/search/mtv.py

```diff
@@ -0,0 +1,235 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+
+import logging
+import datetime
+import re
+
+from sqlalchemy import Column, Unicode, DateTime
+
+from flexget import plugin, db_schema
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.requests import TimedLimiter, RequestException
+from flexget.manager import Session
+from flexget.utils.database import json_synonym
+from flexget.utils.requests import Session as RequestSession
+from flexget.utils.soup import get_soup
+
+log = logging.getLogger('mtv')
+Base = db_schema.versioned_base('mtv', 0)
+
+requests = RequestSession()
+requests.add_domain_limiter(TimedLimiter('morethan.tv', '5 seconds'))  # TODO find out if they want a delay
+
+CATEGORIES = {
+    'Movies': 'filter_cat[1]',
+    'TV': 'filter_cat[2]',
+    'Other': 'filter_cat[3]'
+}
+
+TAGS = [
+    'action',
+    'adventure',
+    'animation',
+    'anime',
+    'art',
+    'asian',
+    'biography',
+    'celebrities',
+    'comedy',
+    'cooking',
+    'crime',
+    'cult',
+    'documentary',
+    'drama',
+    'educational',
+    'elclasico',
+    'family',
+    'fantasy',
+    'film.noir',
+    'filmromanesc',
+    'food',
+    'football',
+    'formula.e',
+    'formula1',
+    'gameshow',
+    'highlights',
+    'history',
+    'horror',
+    'investigation',
+    'lifestyle',
+    'liga1',
+    'ligabbva',
+    'ligue1',
+    'martial.arts',
+    'morethan.tv',
+    'motogp',
+    'musical',
+    'mystery',
+    'nba',
+    'news',
+    'other',
+    'performance',
+    'philosophy',
+    'politics',
+    'reality',
+    'romance',
+    'romanian.content',
+    'science',
+    'scifi',
+    'short',
+    'silent',
+    'sitcom',
+    'sketch',
+    'sports',
+    'talent',
+    'tennis',
+    'thriller',
+    'uefachampionsleague',
+    'uefaeuropaleague',
+    'ufc',
+    'war',
+    'western',
+    'wta'
+]
+
+
+class MTVCookie(Base):
+    __tablename__ = 'mtv_cookie'
+
+    username = Column(Unicode, primary_key=True)
+    _cookie = Column('cookie', Unicode)
+    cookie = json_synonym('_cookie')
+    expires = Column(DateTime)
+
+
+class SearchMTV(object):
+    """
+        MorethanTV search plugin.
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'username': {'type': 'string'},
+            'password': {'type': 'string'},
+            'category': {
+                'oneOf': [
```

Comment:
You can use the `one_or_more` tool for that: https://github.com/Flexget/Flexget/blob/b92f47a265db04e8e4ec8dd2bff7a54b1a6f1581/flexget/config_schema.py#L65-L65